docs/anchoring-assets/anchor-platform/sep24/integration.mdx

```diff
@@ -67,6 +67,31 @@ SECRET_PLATFORM_API_AUTH_SECRET="your encryption key shared with your business s
 
 After it's enabled, all requests must contain a valid `Authorization` header. The JWT provided must have the `jti` and `exp` fields representing a valid transaction and token expiration time, respectively.
 
+## Working With User Request
```

Comment:
I'm not sure this is the right place to add this section. The introduction to the "Integration" section only talks about `PATCH /transactions` requests, so adding content on the JWT and `/fee` endpoint seems out of place. Maybe we should move this section to the bottom, or add new FAQ page?

Reply:
I can move it to a new section, WDYT?